app/views/components/_result-card.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  title ||= nil
+  description ||= nil
+  url ||= nil
+  url_text ||= nil
+%>
+<div class="app-c-result-card">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: title,
+    heading_level: 2,
+    margin_bottom: 2,
+    font_size: "m"
+  } %>
+
+  <div class="app-c-result-card__description">
+    <p class="govuk-body">
+      <%= description %>
+    </p>
+  </div>
+
+  <div class="app-c-result-card__link">
+    <p><%= link_to url_text, url, class: "govuk-link" %></p>
+  </div>
```

Comment:
Could we remove the need for a wrapping `div` and `p` elements by having the two classes on the same element?
```suggestion
    <%= link_to url_text, url, class: "govuk-link app-c-result-card__link" %>
```

Reply:
Thanks, I have made this change and removed the `div` and `p` elements. I tested it locally and still renders as expected.